modules/dynamic-plugins/proc-export-third-party-plugins-rhdh.adoc

```diff
@@ -0,0 +1,124 @@
+[id="proc-export-third-party-plugins-rhdh_{context}"]
+= Exporting third-party plugins in {product}
+
+To use plugins in {product}, you can export plugins as derived dynamic plugin packages. These packages contain the plugin code and dependencies, ready for dynamic plugin integration into {product-short}.
+
+Backend plugins::
++
+--
+To ensure compatibility with the dynamic plugin support and enable their use as dynamic plugins, existing backend plugins must be compatible with the new backend system. Additionally, these plugins must be rebuilt using a dedicated CLI command.
+
+The new backend system entry point (created using `createBackendPlugin()` or `createBackendModule()`)  must be exported as the default export from either the main package or an `alpha` package (if the plugin instance support is still provided using `alpha` APIs). This doesn't add any additional requirement on top of the standard plugin development guidelines of the plugin instance.
```

Comment:
```suggestion
The backend system entry point (created using `createBackendPlugin()` or `createBackendModule()`)  must be exported as the default export from either the main package or an `alpha` package (if the plugin instance support is still provided using `alpha` APIs). This requirement aligns with the standard plugin development guidelines and does not introduce any additional rules for plugin instances.
```
I'm not sure we need to keep referring to the backend system as "new", unless something has changed in this release.

The other suggested change here is just cosmetic but I think it reads ok.

Reply:
@Gerry-Forde Good point.
@kadel Should we continue referring to it as the "new backend system," or would it be better to simply call it the "backend system"? By version 1.5, the "new" backend system may no longer be considered new, so this adjustment might be helpful for maintenance purposes.

Reply:
It should be definitely a new backend system. This is how it is known by developers